ios/RNPurchases.m

```diff
@@ -85,21 +86,23 @@ - (dispatch_queue_t)methodQueue
                  purchaseProduct:(NSString *)productIdentifier
                  upgradeInfo:(NSDictionary *)upgradeInfo
                  type:(NSString *)type
+                 signedDiscountTimestamp:(NSString *)signedDiscountTimestamp
                  resolve:(RCTPromiseResolveBlock)resolve
                  reject:(RCTPromiseRejectBlock)reject)
 {
-    [RCCommonFunctionality purchaseProduct:productIdentifier completionBlock:[self getResponseCompletionBlockWithResolve:resolve reject:reject]];
+    [RCCommonFunctionality purchaseProduct:productIdentifier signedDiscountTimestamp:signedDiscountTimestamp completionBlock:[self getResponseCompletionBlockWithResolve:resolve reject:reject]];
```

Comment:
nitpick: breaking line between params here makes it easier to read without horizontal scrolling

Reply:
same for other similar calls